docs/deploy/configurations.md

```diff
@@ -127,6 +127,16 @@ Once disabled:
 * you can no longer create Ingresses with the `alb.ingress.kubernetes.io/group.name` annotation.
 * you can no longer alter the value of an `alb.ingress.kubernetes.io/group.name` annotation on an existing Ingress.
 
+### sync-period
+`--sync-period` defines a fix interval for the controller to auto reconcile, default to 1hr. Please be mindful that frequent reconciliations may incur unnecessary AWS API usage.
+
+As best practice, we do not recommend users to manually modify the resources managed by the controller. And users should not depend on the controller auto-reconciliation to revert the manual modification, or to mitigate any security risks. 
+Since no matter how frequent the controller reconciles, there should always be a gap.
```

Comment:
remove this line `Since no matter how frequent the controller reconciles, there should always be a gap.`